lib/homepage_options/choose_option.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+
+class ChooseOption extends StatelessWidget {
+  void chooseStudent() {
+    //Navigation to student
+  }
+
+  void chooseDocent() {
+    //Navigation to teacher
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        ElevatedButton(
+            onPressed: chooseStudent,
+            child: Text(
+              "Student",
+              style: new TextStyle(fontSize: 30.0),
+            ),
+            style: ElevatedButton.styleFrom(
+                fixedSize: const Size(240, 80), primary: Colors.red)),

```

Comment:
```suggestion
                fixedSize: const Size(240, 80), primary: Colors.red),),

```
Als je komma hier plaatst en bestand opslaat, zal u code automatisch formateren en niet meer rare dubbele identantie krijgen ;)